packages/postgres-database/src/simcore_postgres_database/models/users.py

```diff
@@ -80,19 +73,29 @@ class UserStatus(Enum):
         "id",
         sa.BigInteger,
         nullable=False,
-        doc="Primary key for user identifier",
+        doc="Primary key index for user identifier",
     ),
     sa.Column(
         "name",
-        sa.String,
+        sa.String(),
         nullable=False,
-        doc="Display name. NOTE: this is NOT a user name since uniqueness is NOT guaranteed",
+        doc="username is a unique short user friendly identifier e.g. pcrespov, sanderegg, GitHK, ...",
+    ),
+    sa.Column(
+        "first_name",
+        sa.String(50),
```

Comment:
A question or more an observation. Why limit the name and surname to 50. Of all the fields that can be limited I would limit the `name` since usually a nickname is shorter than one's name or surname (especially for people with long names and surnames)

Reply:
This was a recommendation that I read and als "chatty" did ... 
- Note that the rest API is already limiting to 50 characters therefore it should fit. If we have a user that wants more, we can extend it later
- I will limit the `username`